pubsub/README.rst

```diff
@@ -41,6 +39,88 @@ Here's the rough usage pattern for using ``SubscriberClient``:
         max_messages=10)
 
 
+There's also ``gcloud.aio.pubsub.subscribe`` helper function you can use to
+setup a pubsub processing pipeline. It is built with ``asyncio`` and thus only available
+in ``gcloud-aio-pubsub`` package. The usage is fairly simple:
+
+.. code-block:: python
+
+    from gcloud.aio.pubsub import SubscriberClient
+    from gcloud.aio.pubsub import subscribe
+    from gcloud.aio.pubsub.metrics_agent import MetricsAgent
+
+    subscriber_client = SubscriberClient()
+
+    async def handler(message):
+        return
+
+    await subscribe(
+        'projects/<my_project>/subscriptions/<my_subscription>',
+        handler,
+        subscriber_client,
+        num_workers=1,
+        max_messages=100,
+        ack_window=0.3,
+        consumer_pool_size=1,
+        metrics_client=MetricsAgent()
+    )
+
+While defaults are somewhat sensinble, it is highly recommended to performance test
+your application and tweak function parameter to your specific needs. Here's a few hints:
+
+:``handler``:
+    an async function that will be called for each message. It should accept an instance of
+    ``SubscriberMessage`` as its only argument and return ``None`` if the message should
+    be acked.
```

Comment:
Add something like "The message will be nacked if the handler raises an exception"?

Reply:
psst, Kevin, it won't ba acked in the current version, it will be left to expire...

Reply:
I think emphasizing this is also important. i.e. adding something like "An exception raised within the handler will result in the message being left to expire, and thus it will be redelivered according to your subscription's ack deadline"

Reply:
makes a lot of sense, thanks!